lib/api.js

```diff
@@ -48,6 +49,40 @@ function removeCoupling(heroku, app) {
            .then(coupling => deleteCoupling(heroku, coupling.id));
 }
 
+function getPipelineCouplings(heroku, pipelineId) {
```

Comment:
1. Drop the `Pipeline` infix. The other wrappers here don't include it.
2. Should we call it `getCouplings` or `listCouplings`?